packages/xdl/src/Project.js

```diff
@@ -1892,6 +1894,38 @@ async function stopWebpackServerAsync(projectRoot) {
   });
 }
 
+export async function bundleWebpackAsync(projectRoot, packagerOpts) {
+  let { exp } = await ProjectUtils.readConfigJsonAsync(projectRoot);
+  if (!exp.platforms.includes('web')) {
```

Comment:
Perhaps this should throw and error that explains how to enable `web` in app.json? Presumably people want to build for web if they run `expo build:web`, so it would be better to show an error than just not do anything.